### PR TITLE
magic mirror no longer shows standard makeup window

### LINKED ISSUE #7255: magic mirror shouldn't call 'normal mirror's appearance change proc
## Round ID:
39223

## Reproduction:

When you use a magic mirror (pride, or wizard den, clock cult), you see a normal mirror appearance change.
this menu shouldn't pop up when you use a magic mirror.

### PR DESCRIPTION
**Pull request by EvilDragonfiend**
## About The Pull Request
Fixes that magic mirror shows non-magical makeup window

## Why It's Good For The Game

* fixes #7255

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

it no longer shows annoying non-magical selection window

</details>

## Changelog
:cl:
fix: magic mirror no longer shows non-magical makeup selection window
/:cl:

### CODE REVIEW

**Review by Penwin0 — CHANGES_REQUESTED** [review]
You can now use broken magic mirrors, you should not be able to do this

**Comment by EvilDragonfiend** [comment]
> You can now use broken magic mirrors, you should not be able to do this

It's been possible to use. The in-game fix for this annoying magic mirror's non-magical .essage was to break magic mirror.

File: code/game/objects/structures/mirror.dm [hunk 1]
```diff
@@ -23,7 +24,7 @@
 	if(broken || !Adjacent(user))
 		return
 
-	if(ishuman(user))
+	if(ishuman(user) && !magical)
```
Comment (Archanial) [inline code/game/objects/structures/mirror.dm:27]: can't you just not call parent in magic window?

Comment (EvilDragonfiend) [reply code/game/objects/structures/mirror.dm:27]: it is supposed to call its parent's parent, so, not calling its parent doesn't look like a good idea - unless `...()` is a thing.

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**